read-the-docs-2021-stats.rst

```diff
@@ -0,0 +1,122 @@
+.. post:: Mar 17, 2022
+   :tags: stats, year-in-review
+
+Read the Docs 2021 Stats
+========================
+
+2021 continued in the realm of being a tough year.
+That said,
+Read the Docs had a lot of good things happen this year.
+
+We did a majority of the owrk on our :doc:`CZI grant </czi-grant-announcement>`,
+grew our team from 5 to 8,
+and continued to grow our `EthicalAds network`_.
```

Comment:
We mention EthicalAds growth, but don't mention commercial at all. It's probably worth noting up front, to communicate commercial hosting is as strong a priority as community and ethicalads.